policyengine_uk/variables/contrib/labour/private_school_vat.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_uk.model_api import *
+from sys import stderr
+
+
+class private_school_vat(Variable):
+    label = "Private school VAT"
+    entity = Household
+    definition_period = YEAR
+    value_type = float
+    unit = "currency-GBP"
+
+    def formula(household, period, parameters):
+        private_school_vat_rate: float = parameters(
+            period
+        ).gov.contrib.labour.private_school_vat
+        household_income_decile: int = household(
+            "household_income_decile", period
+        )
+        attends_private_school: bool = (
+            random(household)
+            < parameters(
+                period
+            ).calibration.programs.private_school_vat.private_school_attendance_rate[
+                household_income_decile
+            ]
+        )
```

Comment:
CODE IN

percent_in_private_school_by_percentile = [0, 0, 0, ..., 0.45, 0.5]
ADJUSTMENT_FACTOR = 1
adjusted_percent_in_private_school_by_percentile *= ADJUSTMENT_FACTOR

# calculate household income percentile
# map to percent in private school
# put kids in private school according to that percentage (mostly done here except percentile not decile)
...
TESTING_CODE
is_child = [False, False, True, False, ...]
child_household_weight = [500, 1000, 2000, 700, ...]
is_in_private_school = [False, False, True, False, ...]
(is_in_private_school * child_household_weight).sum()

model_predicted_kids_in_private_school = 800k # should be 600k